### PR TITLE
OCPNODE-638: Kubelet-Controller Manager communication Profiles (WorkerLatencyProfiles)

### PR DESCRIPTION
**Pull request by mburke5678**
https://issues.redhat.com/browse/OSDOCS-3045

Previews: 
Nodes -> Clusters -> [Improving cluster stability in high latency environments using worker latency profiles](http://file.rdu.redhat.com/mburke/nodes-workerlatencyprofiles.html) 

Post-Install -> Cluster tasks -> [Improving cluster stability in high latency environments using worker latency profiles](http://file.rdu.redhat.com/mburke/cluster-tasks.html#post-worker-latency-profiles)

### CODE REVIEW

**Comment by mburke5678** [comment]
@harche Can you review my PR for me, or suggest someone who can? Thank you in advance!

**Comment by harche** [comment]
> @harche Can you review my PR for me, or suggest someone who can? Thank you in advance!

Thanks @mburke5678. I will take a look at this one on coming Monday after the shift week is over.

File: modules/nodes-cluster-worker-latency-profiles-about.adoc [hunk 1]
```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+//
+// * nodes/clusters/nodes-cluster-worker-latency-profiles
+// * nodes/edge/nodes-edge-remote-workers. ??
+// * post_installation_configuration/cluster-tasks ??
```
[the remark this answers is not in the document]
Comment (mjpytlak) [reply modules/nodes-cluster-worker-latency-profiles-about.adoc:5]: General comment for all modules. I understand that these are possible placeholders, but ahead of a merge, please be sure to only specify the assemblies in which the module is included.

**Review by mjpytlak — COMMENTED** [review]
Left some feedback for your review @mburke5678.

**Pull request merged**